editor/editor_file_system.cpp

```diff
@@ -2611,6 +2611,11 @@ void EditorFileSystem::reimport_files(const Vector<String> &p_files) {
 
 	EditorProgress pr("reimport", TTR("(Re)Importing Assets"), p_files.size());
 
+	const bool old_low_processor_usage_mode = OS::get_singleton()->is_in_low_processor_usage_mode();
+	const DisplayServer::VSyncMode old_vsync_mode = DisplayServer::get_singleton()->window_get_vsync_mode(DisplayServer::MAIN_WINDOW_ID);
+	OS::get_singleton()->set_low_processor_usage_mode(false);
+	DisplayServer::get_singleton()->window_set_vsync_mode(DisplayServer::VSyncMode::VSYNC_DISABLED);
```

Comment:
There should be a comment above to explain why this is done, as it's not obvious at first.

Reply:
Done!